thirdparty/patches/opentelemetry-cpp-1.2.0.patch

```diff
@@ -0,0 +1,374 @@
+diff --git a/api/include/opentelemetry/common/threadlocal.h b/api/include/opentelemetry/common/threadlocal.h
```

Comment:
😂 you git push the `patch` file.

Reply:
This patch is to solve the problem that the opentelemtry-cpp library depends on `GLIBC_2.18`, and replaces the `thread_local` keyword in the opentelemtry-cpp library.